kubernetes/ansible/roles/sunbird-monitoring/defaults/main.yml

```diff
@@ -220,3 +221,21 @@ redis_process_names: ""
 
 vault_azure_spark_cluster_http_password: ""
 azure_spark_cluster_http_password: "{{ vault_azure_spark_cluster_http_password }}"
+
+service_health_checks:
+  - service_name: 'analytics'
+    targets: "http://analytics-service.{{ namespace }}.svc.cluster.local:9000/health"
+  - service_name: 'learning-service'
+    targets: "http://{{ learningservice_ip }}:8080/learning-service/healthy"
```

Comment:
@Kaali09 are you sure it's /healthy ?

Reply:
It should be /health. updated the code. please merge this https://github.com/project-sunbird/sunbird-devops/pull/2346